README.md

```diff
@@ -72,4 +72,4 @@ Ballerina code is distributed under [Apache license 2.0](https://github.com/ball
 
 * The ballerina-dev@googlegroups.com mailing list is for discussing code changes to the Ballerina project.
 * Chat live with us on our [Slack channel](https://ballerina-platform.slack.com/).
-* User support should be posted on Stack Overflow with the #ballerina tag.
+* Technical questions should be posted on Stack Overflow with the [#ballerinalang](https://stackoverflow.com/questions/tagged/ballerinalang) tag.
```

Comment:
#ballerinalang -> #ballerina

Reply:
Correct tag is #ballerinalang

Reply:
Fixed.

Reply:
Hi Abimaran, 

Can you reword your commit message? "corrected" -> "correct"